wallet/chain/p/builder.go

```diff
@@ -51,16 +51,14 @@ type Builder interface {
 		options ...common.Option,
 	) (map[ids.ID]uint64, error)
 
-	// NewBaseTx creates a new simple value transfer. Because the P-chain
-	// doesn't intend for balance transfers to occur, this method is expensive
-	// and abuses the creation of subnets.
+	// NewBaseTx creates a new simple value transfer.
```

Comment:
I'm a bit concerned that we are breaking this method prior to the D-upgrade activating.

Reply:
reverted and added to https://github.com/ava-labs/avalanchego/issues/1965 for tracking